dll/win32/shell32/wine/control.c

```diff
@@ -736,6 +736,53 @@ static	void	Control_DoWindow(CPanel* panel, HWND hWnd, HINSTANCE hInst)
 }
 #endif
 
+/** Structure for in and out data when
+ *  search for the cpl dialog of first instance
+ */
+typedef struct tagAppDlgFindData
+{
+    const WCHAR*    szCaption;  /**< second search value */
+    HWND            hResult;    /**< variable for result value */
```

Comment:
```suggestion
    PCWSTR pszCaption;  /**< second search value */
    HWND   hDlgResult;  /**< variable for result value */
```
Descriptions could be a bit more explicit: for example,
"Title of dialog to search for", and "Returned dialog handle (or NULL if not found)."

Reply:
Okay